data/lib/core/player.lua

```diff
@@ -384,6 +384,80 @@ function Player.setSpecialContainersAvailable(self, available)
 	return true
 end
 
+function Player.getMaxTrackedQuests(self)
+	return configManager.getNumber(self:isPremium() and configKeys.QUEST_TRACKER_PREMIUM_LIMIT or configKeys.QUEST_TRACKER_FREE_LIMIT)
+end
+
+function Player.sendQuestLog(self)
+	local msg = NetworkMessage()
+	msg:addByte(0xF0)
+	local quests = Quest.getQuests(self)
```

Comment:
`getQuests` does not return all quests so it should be clear about it. `getPlayerQuests`

Also, I think it should be in the `Game` or `Player` namespace, not `Quest`